src/kpl_agg.erl

```diff
@@ -151,16 +151,6 @@ add_all(State, Records) ->
 %%% Internal functions
 %%%===================================================================
 
-%% Calculate a new explicit hash key based on the input partition key
-%% (following the algorithm from the original KPL).
-create_explicit_hash_key(_PartitionKey) ->
-    %% Their python implementation [1] is broken compared to the C++
-    %% implementation [2]. But we don't care about EHKs anyway.
-    %% [1] https://github.com/awslabs/kinesis-aggregation/blob/db92620e435ad9924356cda7d096e3c888f0f72f/python/aws_kinesis_agg/aggregator.py#L447-L458
-    %% [2] https://github.com/awslabs/amazon-kinesis-producer/blob/ea1e49218e1a11f1b462662a1db4cc06ddad39bb/aws/kinesis/core/user_record.cc#L36-L45
-    %% FIXME: Implement the actual algorithm from KPL.
```

Comment:
._. shouldn't we actually create a ticket to fix this?

Reply:
After all these years?
I think it is very clear that it works well as it is.